website/src/content/deployment-examples/aws-s3-cloudfront.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Deploying to AWS with S3 and CloudFront
+---
+
+This deployment example refers to [AWS](https://aws.amazon.com/) static hosting using an [S3](https://s3.console.aws.amazon.com/s3/home) bucket and [CloudFront](https://console.aws.amazon.com/cloudfront/home) distribution.
+
+# Prerequisites
+
+Before you get started, you need to have:
+
+- An [AWS](https://portal.aws.amazon.com/billing/signup#/start) account.
+- A [commercetools](https://docs.commercetools.com/getting-started) account and a Project.
+- A Custom Application [configured in the Merchant Center](https://docs.commercetools.com/merchant-center/managing-custom-applications).
+
+# Configuration
+
+In your Custom Application config, make sure to provide the [Custom Application ID](/api-reference/application-config#envproductionapplicationId) that you got when you configured the Custom Application in the Merchant Center.
+
+Moreover, you need to provide the [production URL](/api-reference/application-config#envproductionurl) from your Netlify site. You can keep the standard Netlify URL `https://<site>.netlify.app` or provide your own custom domain.
```

Comment:
Here we're mentioning Netlify when these are the AWS docs.

Reply:
Good catch!